admin/config-ui/fields/class-field-multiple-authors.php

```diff
@@ -67,14 +67,15 @@ public function get_data() {
 	public function set_data( $data ) {
 		$value = ( $data === 'yes' );
 
-		$option                           = WPSEO_Options::get_option( 'wpseo' );
-		$option['has_multiple_authors'] = $value;
+		// Set multiple authors.
+		$resultMultipleAuthors = WPSEO_Options::save_option( 'wpseo', 'has_multiple_authors', $value );
```

Comment:
Make sure you follow the WordPress codestyle. I know it's annoying compared to the JS codes where camelUppercase should be used. But in WordPress codestyle we use `_` between the words.